app/connections/clean_connection.rb

```diff
@@ -0,0 +1,23 @@
+# This stands in for ActiveFedora::CleanConnection. It behaves the same way,
```

Comment:
I think @tpendragon wrote the original in ActiveFedora so that a resource could be displayed in the catalog controller.  In that case he didn't want to expose has_model and he didn't need it to figure out the route (all route to `/catalog/{id}`)